es6.md

```diff
@@ -89,3 +89,52 @@ interface ArrayPattern <: Pattern {
     elements: [ Pattern | null ];
 }
 ```
+
+# Classes
+
+```js
+interface Class <: Node {
+    id: Identifier | null;
+    superClass: Expression;
+    body: ClassBody;
+}
+```
+
+## ClassBody
+
+```js
+interface ClassBody <: Node {
+    type: "ClassBody";
+    body: [ MethodDefinition ];
+}
+```
+
+## MethodDefinition
+
+```js
+interface MethodDefinition <: Node {
+    type: "MethodDefinition";
+    key: Identifier;
+    value: FunctionExpression;
+    kind: "" | "get" | "set";
```

Comment:
They are [the same in the spec however](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-method-definitions), that's why I was wondering if it makes sense to treat them the same in an AST as well.